libraries/AP_GPS/AP_GPS_Logging.cpp

```diff
@@ -0,0 +1,54 @@
+#include "AP_GPS.h"
+
+#include <AP_Logger/AP_Logger.h>
+
+// Write an GPS packet
+void AP_GPS::Write_GPS(uint8_t i)
+{
+    const AP_GPS &gps = AP::gps();
```

Comment:
Could remove this and all the `gps.` below.

```suggestion
```

Reply:
Done, thanks.

It's possible @WickedShell might bounce this separate patch as I'm no longer just moving code around.